cmd/mercury/main.go

```diff
@@ -20,15 +22,25 @@ func main() {
 	db := kv.NewMemDB(kv.JSONCodec)
 
 	// Initialise stores.
-	ethRinkebyCache := cache.New(kv.NewTable(db, "ethRinkeby"), logger)
-	ethKovanCache := cache.New(kv.NewTable(db, "ethKovan"), logger)
-	btcTestCache := cache.New(kv.NewTable(db, "btcTest"), logger)
-	zecTestCache := cache.New(kv.NewTable(db, "zecTest"), logger)
-	bchTestCache := cache.New(kv.NewTable(db, "bchTest"), logger)
-	ethCache := cache.New(kv.NewTable(db, "eth"), logger)
-	btcCache := cache.New(kv.NewTable(db, "btc"), logger)
-	zecCache := cache.New(kv.NewTable(db, "zec"), logger)
-	bchCache := cache.New(kv.NewTable(db, "bch"), logger)
+	rinkebyTTL := kv.NewTTLCache(context.Background(), db, "ethRinkebyCache", 5*time.Second)
+	kovanTTL := kv.NewTTLCache(context.Background(), db, "ethKovanCache", 5*time.Second)
+	// btcTestTTL := kv.NewTTLCache(context.Background(), db, "btcTestCache", 5*time.Second)
+	// zecTestTTL := kv.NewTTLCache(context.Background(), db, "zecTestCache", 5*time.Second)
+	// bchTestTTL := kv.NewTTLCache(context.Background(), db, "bchTestCache", 5*time.Second)
+	// ethTTL := kv.NewTTLCache(context.Background(), db, "ethCache", 5*time.Second)
+	// btcTTL := kv.NewTTLCache(context.Background(), db, "btcCache", 5*time.Second)
+	// zecTTL := kv.NewTTLCache(context.Background(), db, "zecCache", 5*time.Second)
+	// bchTTL := kv.NewTTLCache(context.Background(), db, "bchCache", 5*time.Second)
```

Comment:
Why don't we use a TTL for these chains?

Reply:
Try to make as minimal change as possible. Don't have enough test suites to ensure these change will work. So I only add the ttl cache for ethereum.